bin/millau/node/src/service.rs

```diff
@@ -69,13 +71,17 @@ pub fn new_partial(
 				AuraPair,
 			>,
 			sc_finality_grandpa::LinkHalf<Block, FullClient, FullSelectChain>,
+			Option<TelemetrySpan>,
 		),
 	>,
 	ServiceError,
 > {
+	if config.keystore_remote.is_some() {
```

Comment:
I was wondering, if we even want to support a remote keystore at all for Rialto and Millau nodes.

Reply:
Definitely not for the testnets, we might consider later on using them for production deployment.